invenio_communities/views/communities.py

```diff
@@ -118,10 +118,19 @@ def communities_settings(pid_value, community, community_ui):
     except FileNotFoundError:
         logo = False
 
+    logo_size_limit = (
+        current_communities.service.config.community_logo_size_default_quota
+    )
+    if "APP_RDM_COMMUNITY_LOGO_QUOTA" in current_app.config:
```

Comment:
This is not ideal because `invenio-communities` should not know about `invenio-app-rdm`. Can we, instead, move the config here?

I would also align the config name with `invenio-files-rest` for consistency, see [here](https://github.com/inveniosoftware/invenio-files-rest/blob/master/invenio_files_rest/config.py#L48). WDYT?
E.g. `COMMUNITIES_LOGO_MAX_FILE_SIZE`.

Reply:
My rationale was whether this quota size could be manageable by instance. If we move the config to the service, how could users change it? I am asking it because I am not 100% familiarised on how we handle this kind of configs in invenio RDM. 

Regarding `invenio-communities` not knowning about `app-rdm`, I think we're using `current_app.config` in many other places in communities. What's the criteria there?

I agree with the naming 👍

Reply:
About `maxStorage` I agree that it can be removed. I took other config as an example but I think it does not apply to this case. Thanks for the catch! 👍 

About tests, I will add them right away.